packages/astro/src/@types/astro.ts

```diff
@@ -1375,6 +1375,93 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		devOverlay?: boolean;
+
+		// TODO review with docs team before merging to `main`
+		/**
+		 * @docs
+		 * @name experimental.i18n
+		 * @type {object}
+		 * @version 3.5.0
+		 * @type {object}
+		 * @description
+		 *
+		 * Configures experimental i18n routing and allows you to specify some customization options.
+		 */
+		i18n?: {
+			/**
+			 * @docs
+			 * @name experimental.i18n.defaultLocale
+			 * @type {string}
+			 * @version 3.5.0
+			 * @description
+			 *
+			 * The default locale of your website/application. This is a required field.
+			 */
+			defaultLocale: string;
+			/**
+			 * @docs
+			 * @name experimental.i18n.locales
+			 * @type {string[]}
+			 * @version 3.5.0
+			 * @description
+			 *
+			 * A list of all locales supported by the website. e.g. `['en', 'fr', 'es']` This list should also include the `defaultLocale`. This is a required field.
```

Comment:
```suggestion
			 * A list of all locales supported by the website (e.g. `['en', 'es', 'pt_BR']`). This list should also include the `defaultLocale`. This is a required field.
```
Fix some punctuation and it's probably good to show that they don't HAVE to be two letter codes.